vagrant/tutorial/Vagrantfile

```diff
@@ -10,35 +10,10 @@ unless Vagrant.has_plugin?("vagrant-reload")
   raise "vagrant-reload plugin is not installed"
 end
 
-=begin
-  The following error may occur when running `vagrant up`, because
-  of a problem with some VirtualBox versions.
-
-    # Failed to mount folders in Linux guest. This is usually because
-    # the "vboxsf" file system is not available. Please verify that
-    # the guest additions are properly installed in the guest and
-    # can work properly. The command you attempted was:
-    #
-    # mount -t vboxsf -o uid=`id -u vagrant`,gid=`getent group vagrant | cut -d: -f3` vagrant /vagrant
-    # mount -t vboxsf -o uid=`id -u vagrant`,gid=`id -g vagrant` vagrant /vagrant
-    #
-    # The error output from the last command was:
-    #
-    # /sbin/mount.vboxsf: mounting failed with the error: No such device
-
-  To help avoid this potential issue, the vagrant-vbguest plugin is required.
-  If it remains, try updating VirtualBox.
-  Known working versions include Vagrant 1.7.2 and VirtualBox 4.3.24r98716.
-=end
-unless Vagrant.has_plugin?("vagrant-vbguest")
-  raise "vagrant-vbguest plugin is not installed"
```

Comment:
Expected to the extent that I've seen that happen and it doesn't seem to cause any problems.
Tom can probably answer this in more detail. FWIW on my local environment, I get

```
Installing additional modules ...
Removing existing VirtualBox DKMS kernel modules[  OK  ]
Removing existing VirtualBox non-DKMS kernel modules[  OK  ]
Building the VirtualBox Guest Additions kernel modules[  OK  ]
Doing non-kernel setup of the Guest Additions[  OK  ]
You should restart your guest to make sure the new modules are actually used

Installing the Window System drivers
Could not find the X.Org or XFree86 Window System, skipping.
An error occurred during installation of VirtualBox Guest Additions 4.3.28. Some functionality may not work as intended.
In most cases it is OK that the "Window System drivers" installation failed.
```